shinyswatch/picker/theme_picker.js

```diff
@@ -225,7 +225,11 @@
     }
   }
 
-  $(window).one("shiny:idle", showWarning)
+  if (typeof window.Shiny.setInputValue === "function") {
+    setTimeout(showWarning, 1000)
```

Comment:
Turns out we need this case if the theme picker is added via dynamic UI but the server isn't registered

Reply:
Thank you!